src/components/Error/Error.jsx

```diff
@@ -36,7 +36,7 @@ const Error = ({ errors, showBackButton, canDismissAlert, router }) => {
 Error.propTypes = {
   canDismissAlert: PropTypes.bool,
   errors: PropTypes.shape({
-    errorText: PropTypes.string.isRequired,
+    errorText: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
```

Comment:
re: [my prior request](https://github.com/scientist-softserv/webstore-component-library/pull/131#discussion_r1084568484)

isn't errorText an [array on strings](https://github.com/scientist-softserv/webstore/blob/main/utils/api/configurations.js#L72)?

Reply:
yes definitely, i was thinking of errors! my bad just changed it